### ISSUE TITLE
[Audio Player]: QA: Only pointer gestures are available for Volume slider

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9286]
### Description

Page Element: Volume slider

Issue Description:
Only pointer gestures are available for Volume slider. No buttons are available to increase/decrease volume

Expected:Buttons should be available for the user to increase/decrease the volume

User Impact:
When only gestures are available, keyboard users who have seizure problems will face difficulty in accessing the application.

### Component(s) impacted

Audio Player

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
Design updates add volume control buttons 
![Screen Shot 2022-08-29 at 11 08 58 AM](https://user-images.githubusercontent.com/45692622/187233691-06e32413-25f3-44ad-8994-a34ce3d60ed0.png)
